### ISSUE TITLE
1Password business?

### ISSUE THREAD

**Issue by jpmurray** [svenluijten/awesome-password-managers#1]
Hey there,

Any opinion on the business versions of stuff? I'm thinking 1Password for business (and even families, actually). Should we include that?

**Comment by svenluijten** [comment]
Yeah I considered this. Maybe every password manager should get its own section instead of just a row in a table. If the table is done away with, we could elaborate more on business / family plans.

**Comment by svenluijten** [comment]
How's [this](https://github.com/svenluijten/awesome-password-managers/tree/section-per-entry)?

**Comment by jpmurray** [comment]
Personally, I like it. But it's true it's less concise... It'd be more work to maintain, but what about a regular "awesome list" that lists passwords, giving links to site, but also redirect to another `service.md` file with more infos? (Just thinking out loud here...)
